### PR TITLE
Update FixedLayoutParser.cpp

### PR DESCRIPTION
**Pull request by kryptonpust**
updated according to hexadecimal value

### CODE REVIEW

File: src/engine/libengine/FixedLayoutParser.cpp [hunk 1]
```diff
@@ -319,7 +319,7 @@ QString FixedLayoutParser::getCharForKey(int keyNum, bool shift, bool altgr, boo
   if (!key.contains("Num"))
   {
     // Check if the key is an alphabet key.
-    if (keyNum >= 41140 && keyNum <= 41165)
+    if (keyNum >= VC_A && keyNum <= VC_Z)
```
Comment (mominul) [inline src/engine/libengine/FixedLayoutParser.cpp]: I was also wanting to do this! :clap:

**Comment by mominul** [comment]
Thanks!

**Comment by mominul** [comment]
Closes #44

**Comment by mominul** [comment]
Closes #44

**Comment by kryptonpust** [comment]
welcome

**Pull request merged**